packages/big-design/src/managers/alerts/manager.ts

```diff
@@ -0,0 +1,88 @@
+import { AlertProps } from '../../components/Alert';
+
+import { Subscriber, TypeMap } from './types';
+
+interface PrivateAlert extends AlertProps {
+  onClose(): void;
+}
+
+class AlertsManager {
+  private alerts: PrivateAlert[] = [];
+  private counter = 0;
+  private subscribers: Subscriber[] = [];
+  private readonly typeMap: TypeMap = {
+    error: 0,
+    warning: 1,
+    success: 2,
+    info: 3,
+  };
+
+  add(alert: AlertProps): string {
+    if (alert.key && this.containsKey(alert.key)) {
+      this.remove(alert.key);
+    }
+
+    const key = alert.key !== undefined ? alert.key : this.getUniqueId();
```

Comment:
🍹 switch the conditions around like so `const key = alert.key === undefined ? this.getUniqueId() : alert.key` for readability